frontend/src/pages/myFarmPage/hooks/useWeeklyWorkSchedule.ts

```diff
@@ -0,0 +1,40 @@
+import dayjs from 'dayjs';
+import { useBaseWeeklyCalendar } from '../../../hooks/useWeeklyCalendar';
+
+export const useWeeklyWorkSchedule = () => {
+  const {
+    currentDate,
+    weekDates,
+    handlePreviousWeek,
+    handleNextWeek,
+    monthLabel,
+  } = useBaseWeeklyCalendar();
+
+  // 현재 주 판단 로직
+  const getWeekLabel = () => {
+    const today = dayjs();
+    const currentWeekStart = dayjs(currentDate).startOf('week');
+    const todayWeekStart = today.startOf('week');
+
+    const weekDiff = currentWeekStart.diff(todayWeekStart, 'week');
+
+    return weekDiff === 0
+      ? '이번 주'
+      : weekDiff < 0
+      ? `${Math.abs(weekDiff)}주 전`
+      : `${weekDiff}주 후`;
+  };
+
+  const weekLabel = getWeekLabel();
+  const isCurrentWeek = getWeekLabel() === '이번 주';
```

Comment:
```suggestion
  const isCurrentWeek = weekLabel === '이번 주';
```
이미 위에 있는 값 같아서 바로 가져다 쓰는 건 어떨까요 ??